docs/UserGuide.md

```diff
@@ -11,8 +11,109 @@ support engineers, and others can validate a cluster’s configuration against a
 
 # Concepts
 
+## Reference Configuration
+
+The target of `oc adm compare` tool is to detect deviations from a live or local set of CRs with respect to a Reference Configuration specified by a Reference Design Solution (RDS) Custom Resources (CRs) recommended by Red Hat.
+
+But first, what do we understand by RDS CRs? The key to setup your RDS CRs is a YAML file called  metadata.yaml which specifies and filters what your RDS CRs looks like.
+
+- The entity metadata descriptor is a YAML file (`metadata.yaml`) defining an RDS
+- A published `metadata.yaml` identifies the set of compliant RDS CR objects. Any new deviation found after running `kubectl cluster compare` tool  shall be identified as a new deviation gap to be worked out by the Red Hat RDS compliance process.  
+- Any incoming customer/partner request to modify the metadata.yaml due to an identified deviation gap shall be brought to the Red Hat RDS compliance process.  RDS compliance process will analyze the impact of the new requested deviation and ultimately intend to bridge the gap between the compliant  and the new identified deviation.
```

Comment:
Not certain in the upstream guide we want to focus purely on the Telco use case, but more generically on how to get the tool, and how to use it against a "reference configuration".

Reply:
ACK